packages/ui/components/ui/block-sync-status/condensed.tsx

```diff
@@ -12,8 +12,9 @@ export const CondensedBlockSyncStatus = ({
   error,
 }: Partial<BlockSyncProps>) => {
   if (error) return <BlockSyncErrorState error={error} />;
+  console.log(latestKnownBlockHeight, fullSyncHeight);
   if (!latestKnownBlockHeight || !fullSyncHeight)
-    return <AwaitingSyncState genesisSyncing={!fullSyncHeight} />;
+    return <AwaitingSyncState genesisSyncing={!!latestKnownBlockHeight} />;
```

Comment:
Minifront incorrectly showed "Genesis state syncing..." instead of "Loading sync state..." for the case when both `fullSyncHeight` and `latestKnownBlockHeight` are undefined